client/signup/processing-screen/index.jsx

```diff
@@ -95,70 +59,40 @@ export class SignupProcessingScreen extends Component {
 	}
 
 	getTitle() {
-		const { loginHandler } = this.props;
-
 		const stepWithDomainItem = find( this.props.steps, step => step.domainItem );
 
 		if ( stepWithDomainItem ) {
 			const domain = stepWithDomainItem.domainItem.meta;
 
-			return loginHandler
-				? this.props.translate(
-						"{{strong}}Done!{{/strong}} Thanks for waiting, %(domain)s is all set up and we're ready {{br/}}for you to get started.",
-						{
-							components: { strong: <strong />, br: <br /> },
-							args: { domain },
-							comment:
-								'The second line after the breaking tag {{br/}} should fit unbroken in 384px and greater and have a max of 30 characters.',
-						}
-				  )
-				: this.props.translate(
-						'{{strong}}Awesome!{{/strong}} Give us one minute and {{br/}}we’ll move right along.',
-						{
-							components: { strong: <strong />, br: <br /> },
-							args: { domain },
-							comment:
-								'The second line after the breaking tag {{br/}} should fit unbroken in 384px and greater and have a max of 30 characters.',
-						}
-				  );
+			return this.props.translate(
+				'{{strong}}Awesome!{{/strong}} Give us one minute and {{br/}}we’ll move right along.',
+				{
+					components: { strong: <strong />, br: <br /> },
+					args: { domain },
+					comment:
+						'The second line after the breaking tag {{br/}} should fit unbroken in 384px and greater and have a max of 30 characters.',
+				}
+			);
 		}
 
-		return loginHandler
-			? this.props.translate(
-					'{{strong}}Done!{{/strong}} Thanks for waiting, we’re ready for you {{br/}}to get started.',
-					{
-						components: { strong: <strong />, br: <br /> },
-						comment:
-							'The second line after the breaking tag {{br/}} should fit unbroken in 384px and greater and have a max of 30 characters.',
-					}
-			  )
-			: this.props.translate(
-					'{{strong}}Awesome!{{/strong}} Give us one minute and {{br/}}we’ll move right along.',
-					{
-						components: { strong: <strong />, br: <br /> },
-						comment:
-							'The second line after the breaking tag {{br/}} should fit unbroken in 384px and greater and have a max of 30 characters.',
-					}
-			  );
+		return this.props.translate(
```

Comment:
That would be neat, but translation strings have to remain strings and not vars.

See: https://github.com/Automattic/i18n-calypso#strings-only

Reply:
Ya, I meant the result of the translate call -- not the input (since it would be the same in both cases).

Reply:
Sorry, I misinterpreted 😇

Reply:
Thanks @jblz for spotting that both translation strings are the same and neither uses the domain. That's another opportunity to remove dead code. We no longer need to figure out the domain name and the `steps` prop becomes unused. Removed it all in 5bb1dca.

The `loader` element turns out to be redundant, too. It's hidden using the `hide-content-accessibly` mixin. And all it says is "Loading...". Removed in e25c023.

The Signup Processing Screen dates back to 2015 (and the pre-OSS repo) and it has been transformed and repurposed many times since. No wonder that so many parts of it no longer make sense.